rrbot_moveit_demo_nodes/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>rrbot_moveit_demo_nodes</name>
+  <version>0.0.0</version>
+  <description>Config and launch files to run rrbot demos with ROS 2, gazebo and moveit2</description>
```

Comment:
Could this package maybe replace `rrbot_gazebo`?

Also, I think you're missing some Gazebo dependencies here

Reply:
@ahcorde , I'm not sure what the answer is. I see you also ported `rrbot_gazebo`, so I guess they serve different purposes. How about documenting both on the README?

Reply:
Improved descriptions https://github.com/ros-simulation/gazebo_ros_demos/pull/41/commits/65fc0709abcbe5216fab69ba1e87802beabfc05a